apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -419,7 +419,7 @@ const (
 	// Its condition status can be one of the following:
 	// - "True" means all the selected resources are successfully applied to all the target clusters.
 	// - "False" means some of them have failed. We will place some of the detailed failure in the FailedResourcePlacement array.
-	// - "Unknown" means we haven't started the apply yet.
+	// - "Unknown" means we haven't started or finished the apply yet.
 	ClusterResourcePlacementAppliedConditionType ClusterResourcePlacementConditionType = "ClusterResourcePlacementApplied"
```

Comment:
now i'm thinking the crp also needs the ClusterResourcePlacementWorkCreated condition